x/foundation/genesis.go

```diff
@@ -10,7 +10,8 @@ import (
 // DefaultGenesisState creates a default GenesisState object
 func DefaultGenesisState() *GenesisState {
 	return &GenesisState{
-		Params: DefaultParams(),
+		Params:               DefaultParams(),
+		OneTimeMintLeftCount: 0,
```

Comment:
Or, you can omit the field.
```suggestion
		Params: DefaultParams(),
```